index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Alecxis</h1>
+    <h2>About</h2>
+    <p>I am a creative Tech/Entrepreneur from Philadelhia, now I am worldwide.</p>
+    <h2>Experience</h2>
+    <li>Portfolio Website, Product Website, Kitchen Designer, Product Designer</li>
```

Comment:
Non blocking since what you have technically meets the assignment requirement but you can also split out each entry into their own lines, something like this:
```
<ul>
  <li>Portfolio Website</li>
  <li>Product Website</li>
  <li>Kitchen Designer</li>
  <li>Product Designer</li>
</ul>
```

You can read more about this here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/li